--- vasp.6.4.2/arch/makefile.include.nvhpc_omp_acc.orig	2024-01-01 22:53:15.401461000 -0500
+++ vasp.6.4.2/arch/makefile.include.nvhpc_omp_acc	2024-01-01 22:54:03.928294531 -0500
@@ -17,8 +17,8 @@
 
 # N.B.: you might need to change the cuda-version here
 #       to one that comes with your NVIDIA-HPC SDK
-FC          = mpif90 -acc -gpu=cc60,cc70,cc80,cuda11.0 -mp
-FCL         = mpif90 -acc -gpu=cc60,cc70,cc80,cuda11.0 -mp -c++libs
+FC          = mpif90 -acc -gpu=cc60,cc70,cc80,cuda12.0 -mp
+FCL         = mpif90 -acc -gpu=cc60,cc70,cc80,cuda12.0 -mp -c++libs
 
 FREE        = -Mfree
 
